osier/technology.py

```diff
@@ -319,7 +346,282 @@ def variable_cost_ts(self, size):
         var_cost_ts : :class:`numpy.ndarray`
             The variable cost time series.
         """
-
-        # assumes single cost
         var_cost_ts = np.ones(size) * self.variable_cost
         return var_cost_ts
+
+
+class RampingTechnology(Technology):
+    """
+    The :class:`RampingTechnology` class extends the :class:`Technology`
+    class by adding ramping attributes that correspond to a technology's
+    ability to increase or decrease its power level at a specified rate.
+
+    Parameters
+    ----------
+    ramp_up_rate : float, :class:`unyt_quantity`
```

Comment:
```suggestion
    ramp_up_rate : float or :class:`unyt_quantity`
```
Change to match other docstrings. Same for line 367